src/main/antora/modules/security/pages/security.adoc

```diff
@@ -1124,3 +1124,216 @@ The `webClient.addRequestHeader()` calls used here make sure that the headers fo
 If you want to inspect the app yourself, you can manually deploy the WAR file (`security/restCustomAuthCustomStore/target/restCustomAuthCustomStore.war` )to the server of your choice (e.g. GlassFish 7), and request the URL via a browser or a commandline util such as `curl`.
 
 
+
+== Securing an endpoint with Form authentication and remember-me
+
+In the following example, we'll secure a REST endpoint using Form authentication and remember-me.
+
+Remember-me is a facility where an authenticated identity can be remembered beyond the scope of an HTTP session.
+
+You'll learn how to:
+
+. Define security constraints
+. Use the Form authentication mechanism
+. Enable the remember-me feature
+. How to define (and implicitly set) a custom remember-me identity store
+. How to define (and implicitly set) a custom identity store
+. Use the Jakarta Security `SecurityContext`
+
+=== Write the application
+
+We'll use the same resource class as we used for <<Securing an endpoint with Form authentication>>
+
+==== Declare the security constraints
+
+We'll use the same security constraints as we used for <<Securing an endpoint with Form authentication>>
+
+==== Declare the authentication mechanism
+
+We'll use the same authentication mechanism declaration as we used for <<Securing an endpoint with Form authentication>>
+
+==== Enable remember-me
+
+In Jakarta Security various services are made available via CDI Interceptors(*), among which the remember-me service. Remember-me can be transparently applied to basically every authentication mechanism. In CDI, it's trivial to add Interceptors to beans that we define ourselves, but a little less trivial to add to provided beans. In this section we explain how to do this via a CDI extension.
+
+[NOTE]
```

Comment:
Please use footnotes: https://docs.asciidoctor.org/asciidoc/latest/macros/footnote/

Reply:
yes, but you can click on the link to go to it, right?